lib/compute/agent-node-config.ts

```diff
@@ -124,13 +125,16 @@ export class AgentNodeConfig {
      });
    }
 
-   public addAgentConfigToJenkinsYaml(stack: Stack, templates: AgentNodeProps[], props: AgentNodeNetworkProps): any {
+   public addAgentConfigToJenkinsYaml(stack: Stack, templates: AgentNodeProps[], props: AgentNodeNetworkProps, macAgent: string): any {
```

Comment:
Shouldn't this be `boolean`? Similar to https://github.com/opensearch-project/opensearch-ci/blob/main/lib/ci-stack.ts#L34

Reply:
Yes its the Same 
The property is passed as Boolean but inferred as string `true` or `false`.

Reply:
https://github.com/opensearch-project/opensearch-ci/blob/1fa8c02d6e78026e8631a87d52a0b7c13c50658e/lib/ci-stack.ts#L46